File: src/app/Copyrights.kt
package app

import react.RBuilder
import react.dom.div
import react.dom.span
// builds Copyrights footer
fun RBuilder.copyrights() =
        div("Copyrights") {
            contentPadding(classes = null) {
                span("white-text") {
                    +"Copyrights © 2020 by Łukasz Huculak"
                }
            }
        }

File: lib/components/copyrights.dart
import 'package:jaspr/html.dart';

class Copyrights extends StatelessComponent {
  @override
  Iterable<Component> build(BuildContext context) sync* {
    yield footer(
      [
        span([text('Created by Łukasz Huculak')]),
      ],
      id: 'Copyrights',
    );
  }
}
